src/action/index.js

```diff
@@ -71,6 +71,14 @@ observe(store, 'unlockerReady', async () => {
   await wallet.init();
 });
 
+/**
+ * Triggered the first time the app was started e.g. to set the
+ * local fiat currency only once.
+ */
+observe(store, 'firstStart', async () => {
```

Comment:
I thought about that too...

> seems like we could get rid of this logic by following the pattern of WalletAction's init function instead (maybe trying to restore from AppStorage instead).

Problem is we can't use the settings action inside wallet init, because setting already depends on wallet i.e. it's injected to fetch the exchange rate.

> or alternately, simplify WalletAction's init function by putting wallet initialization in this trigger..

Not super happy about that either because we can't really unit test this routine. Trying to keep all testable logic in actions.